### PR TITLE
Add new layers for ee-concurrency, servlet and jaxrs-core

### PR DESCRIPTION
**Pull request by jamezp**
JIRA's:

- https://issues.redhat.com/browse/WFLY-13355
- https://issues.redhat.com/browse/WFLY-17804
- https://issues.redhat.com/browse/WFLY-17648
- https://issues.redhat.com/browse/WFLY-17598

### CODE REVIEW

File: ee-feature-pack/galleon-shared/src/main/resources/layers/standalone/web-server/layer-spec.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <dependencies>
         <layer name="naming"/>
         <layer name="ee"/>
+        <layer name="ee-concurrency" optional="true"/>
```
Comment (bstansberry) [inline ee-feature-pack/galleon-shared/src/main/resources/layers/standalone/web-server/layer-spec.xml:6]: You can remove this. You have the 'ee' dep which in turn has the optional dep on ee-concurrency. So you don't need to declare it again.

Comment (jamezp) [reply ee-feature-pack/galleon-shared/src/main/resources/layers/standalone/web-server/layer-spec.xml:6]: Thanks. I knew it looked funny, but couldn't quite figure out why :)

**Review by bstansberry — APPROVED** [review]

File: ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-concurrency.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
+  ~  * JBoss, Home of Professional Open Source.
+  ~  * Copyright $year Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-concurrency.xml:5]: Nitpick: license $year.

File: ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-integration.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
+  ~  * JBoss, Home of Professional Open Source.
+  ~  * Copyright $year Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-integration.xml:5]: Nitpick: license $year.

File: ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-concurrency.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
```
Comment (rhusar) [inline ee-feature-pack/galleon-shared/src/main/resources/feature_groups/ee-concurrency.xml:3]: Nitpick: this license is a Java comment inside a xml comment :-)

**Review by rhusar — CHANGES_REQUESTED** [review]

**Comment by jamezp** [comment]
@rhusar Weird, I wonder what happened there. I'll have to check to see if IDEA changed something or there is a bug. We'll set this as a follow up task as I see other places where `$year` is used.

**Comment by bstansberry** [comment]
@jamezp @rhusar I expect that's my IDE problem, not yours, James.

**Comment by bstansberry** [comment]
Thanks @jamezp @rhusar

**Comment by rhusar** [comment]
> @jamezp @rhusar I expect that's my IDE problem, not yours, James.

Perhaps, what version of Emacs are you running? :-D

**Pull request merged**